docs/dietpi_tools.md

```diff
@@ -424,12 +424,100 @@ It is one of the core tools, enabling you to install or uninstall one or more [*
 
     ![DietPi-CPU_info screenshot](assets/images/dietpi-cpuinfo.jpg)
 
-=== "DietPi survey"
+=== "DietPi Survey"
 
-    This option will upload information on a anonymous base. The aggregated results of all uploads can be seen on the [DietPi survey statistics website](https://dietpi.com/survey/).  
-    Run `dietpi-survey`.
+    DietPi Survey allows the DietPi project to obtain general information regarding your system and installed software.
 
-    ![DietPi-Survey screenshot](assets/images/dietpi-survey.jpg)
+    ![DietPi Survey screenshot](assets/images/dietpi-survey.jpg)
+
+    #### Privacy and goals
+
+    The following privacy rules and goals are considered:
+
+    - No private data is sent. No one can identify you. No IP address is obtained.
+    - Basically DietPi Survey is an ***opt OUT*** system. By selecting ***opt IN***, you are helping us.
+    - The transmitted data allows the DietPi project team to achieve the best future possible experience for everyone to
+        - focus and improve especially popular areas,
+        - ensure the most common devices and software titles will receive support and improvements,
+        - keep up support for software and hardware that you use.
+
+    #### Data transmission events
+
+    The DietPi Survey data file is sent when you install software with `dietpi-software` and update DietPi with `dietpi-update`.
+
+    #### Uploaded amount of data
+
+    The uploaded data is tiny (about 1 KB) and won't effect your internet bandwidth or system performance.
```

Comment:
It still says effect, instead of affect.

Reply:
Oh, thanks. I misunderstood your remark. Of course, affect is right.